content/en/docs/Testing Policies/_index.md

```diff
@@ -50,44 +50,50 @@ The following is an example of a simple GitHub Actions workflow which may be use
 The repository contains Kyverno policies stored in `policies/`, Kubernetes resource manifests stored in `resources/` and complete Kyverno test cases stored in `tests/`. A sample workflow may be added at the path `.github/workflows/kyverno.yaml` with the below contents which performs the following for both opened pull requests and manually if triggered on the repository.
 
 1. The repository is checked out.
-2. The Kyverno CLI is downloaded by naming a specific version. This can be changed to simulate the effect a Kyverno upgrade may have in a cluster.
-3. Tests the unknown manifests in `resources/` against the known policies stored in `policies/`.
-4. Tests the pre-defined test cases stored in `tests/` which contain Kyverno test manifests, policies, and resources separated by folder.
+2. The GitHub action for Kyverno CLI is downloaded by naming a specific version. This can be changed to simulate the effect a Kyverno upgrade may have in a cluster. Please refer to [kyverno-cli-installer](https://github.com/marketplace/actions/kyverno-cli-installer) for more information on the GitHub action for Kyverno CLI.
+3. Check the Kyverno CLI version
+4. Tests the unknown manifests in `resources/` against the known policies stored in `policies/`.
+5. Tests the pre-defined test cases stored in `tests/` which contain Kyverno test manifests, policies, and resources separated by folder.
 
 ```yaml
 name: kyverno-policy-test
 on:
   - pull_request
   - workflow_dispatch
-env:
-  VERSION: v1.7.2
 jobs:
   test:
     runs-on: ubuntu-latest
+    permissions:
+      contents: read
     steps:
       - name: Checkout repo
-        uses: actions/checkout@v3
-      - name: Download Kyverno CLI
-        run: |
-          curl -sLO https://github.com/kyverno/kyverno/releases/download/${{ env.VERSION }}/kyverno-cli_${{ env.VERSION }}_linux_x86_64.tar.gz
-          tar -xf kyverno-cli_${{ env.VERSION }}_linux_x86_64.tar.gz
-          ./kyverno version
+        uses: actions/checkout@v4
+      - name: Install Kyverno CLI
+        uses: kyverno/action-install-cli@v0.2.0
+        with:
+          release: 'v1.9.5'
+      - name: Check install
+        run: kyverno version
       - name: Test new resources against existing policies
-        run: ./kyverno apply policies/ -r resources/
+        run: kyverno apply policies/ -r resources/
       - name: Test pre-defined cases
-        run: ./kyverno test tests/
+        run: kyverno test tests/
 ```
 
 Upon pull request to the repository containing this file, the GitHub Actions workflow will be triggered. If all tests succeed, the output will show a success for all steps.
 
-![ci-pass](/images/ci-pass.png)
+![ci-pass][/images/ci-pass.png]
```

Comment:
Not sure why you're changing the format of these links when they work as-is. This is standard Markdown.